### ISSUE TITLE
Finetune a model like training from scratch, why???

### ISSUE THREAD

**Issue by brisker** [facebook/fb.resnet.torch#150]
Hi, after I trained a model from scratch, I found that the epoch seems not enough, so I reload the model and try to finetune it for a bit more epoches. But why the training loss seems like training from scratch?? I just use this code:
model=torch.load(...)
local parameters, gradParameters = model:getParameters()
...
The other codes are just the same as the code for training from scratch. Why this happens? I am really confused about this.

**Comment by meliketoy** [comment]
Maybe it's because of the high initial learning rate. Try dropping the learning rate significantly while fine tuning.

**Comment by colesbury** [comment]
As @meliketoy suggested the default learning rate is probably too high for fine tuning. Try something closer to 0.01 or maybe even 0.001
